pkg/agent/agent.go

```diff
@@ -288,10 +316,6 @@ func (a *Agent) attest() (map[string]*common.RegistrationEntry, error) {
 		}
 		attestor := plugins[0]
 
-		attestorInfo := a.Catalog.Find(attestor.(common_catalog.Plugin))
-		a.config.Log.Info("Preparing to attest this node against ", a.config.ServerAddress.String(),
-			" using strategy '", attestorInfo.Config.PluginName, "'")
```

Comment:
Was this intentional? It was recently committed to fix https://github.com/spiffe/spire/issues/213

Reply:
not at all intentional..I think this was some conflict resolution issue..will fix it